README.md

```diff
@@ -1,2 +1,57 @@
 embulk-util-config
 ===================
+
+How to release
+---------------
+
+See also: https://central.sonatype.org/pages/gradle.html
+
+### Sonatype
+
+1. Create your account on Sonatype OSSRH.
+2. Set your account information in your `gradle.properties` (usually in `~/.gradle/gradle.properties`).
+    * ```
+      ossrhUsername=<your Sonatype OSSRH username>
+      ossrhPassword=<your Sonatype OSSRH password>
+      ```
+
+### OpenPGP
+
+1. Create your own OpenPGP key pair.
+2. Set your OpenPGP key information in your `gradle.properties`.
+    * ```
+      # For example: signing.keyId=1234ABCD
+      signing.keyId=<your OpenPGP key ID>
+      signing.password=<your OpenPGP key password>
+      # For example: signing.secretKeyRingFile=/home/you/.gnupg/1234567890ABCDEF1234567890ABCDEF12345678.secring.gpg
+      signing.secretKeyRingFile=<path to your OpenPGP secret key ring file>
+      ```
+
+### Release
+
+1. Create a detached `HEAD` from the `master` branch.
+    * ```
+      git checkout master
+      git checkout --detach
```

Comment:
Just curious. Is it really necessary to detach? Having all this in `master` branch doesn't seem to hurt.

Reply:
Sure, I don't mind. Just new for me.